packages/commonwealth/client/scripts/views/pages/discussions/CommentTree/CommentTree.tsx

```diff
@@ -64,6 +64,29 @@ export const CommentTree = ({
   commentSortType,
   disabledActionsTooltipText,
 }: CommentsTreeAttrs) => {
+  const urlParams = new URLSearchParams(location.search);
+  const focusCommentsParam = urlParams.get('focusComments') === 'true';
+
+  useEffect(() => {
+    let timeout;
+
+    if (focusCommentsParam) {
+      timeout = setTimeout(() => {
+        const ele = document.getElementsByClassName('CommentsTree')[0];
+        ele.scrollIntoView({
+          behavior: 'smooth',
+          block: 'start',
+        });
+      }, 100);
+    }
+
+    return () => {
+      timeout !== undefined && clearTimeout(timeout);
+    };
+    // we only want to run this on first mount
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
focusCommentsParam won't change over the time so passing it to the dependency array should be safe

Reply:
actually, there are some manual redraws that are triggered on this page by unrelated logic to this comment focus. Adding to the dep array causes it to run on redraws which is prevented by running on mount.